src/core/config/tabular_data/input_tables/option.h

```diff
@@ -0,0 +1,11 @@
+// This option is meant for the case where an algorithm accepts several
+// tables.
```

Comment:
I added the comment to input_table/option.h so it wouldn't accidentally be used if an algorithm uses 2 tables, for example. But I don't see how this option could be confusing, so this comment isn't needed.

Reply:
Ack